frontend/src/metabase/entities/fields.js

```diff
@@ -86,6 +87,13 @@ const Fields = createEntity({
       return { id, values };
     }),
 
+    updateField(field, opts) {
```

Comment:
I assume `opts` would likely be `undefined` but I put it here anyway since other entities seems to always pass along `opts`.